web/src/main/resources/application.properties.EXAMPLE

```diff
@@ -18,20 +18,12 @@ vep.isoform.overrides=uniprot:isoform_overrides_uniprot.txt,mskcc:isoform_overri
 # PDB header web service URL
 pdb.header_service_url=http://files.rcsb.org/header/PDB_ID.pdb
 
-# Ensembl biomart PFAM filename
-ensembl.biomart.pfam=ensembl_biomart_pfam.txt
```

Comment:
We may still want to keep these file names, and replace the hard-coded versions in the `GenomeNexusImporter`.

Reply:
@onursumer I'm still not sure using `GenomeNexusImporter` is the right way to initialize the database. Making it configurable at this point seems like premature optimization